### PR TITLE
feat: add next button in tab sequence at the end of the verse

### LINKED ISSUE #189: Automatic advance to next verse
When I save the last gloss in a verse it doesn't automatically take me to the next verse. 
So, if a user has already approved/saved all the glosses in a verse, and they hit Enter/Tab on the last word in the verse, it advances automatically to the next verse.
That's definitely an important feature to have before people start using it, since everyone is used to Sheldon's app doing that.

### PR DESCRIPTION
**Pull request by arrocke**
## What has changed

* Added a button and the end of the verse to make it quick to navigate to the next verse after tabbing through each word

## Connected Issues

closes #189 

## QA Steps

Tab through a verse in Hebrew and Greek and then press enter when on the next button.

## Post-Deployment

- [x] Nothing required

### CODE REVIEW

**Comment by arrocke** [comment]
I still have some work to do to ensure the next button is always aligned with the gloss autocomplete

**Review by Pertempto — COMMENTED** [review]
When you use the next button, the keyboard focus on the new verse remains on the next button. I would expect it to go to the first word of the new verse, otherwise that's really going to interrupt the flow of glossing.

A few things I would change about the design:
 - There should be more space between the icon and the text.
     - Maybe this is just a personal preference? I feel like a lot of our icons are "squished" throughout the app.
 - It would feel much more natural to have the next button vertically aligned with the gloss inputs.
     - I could be wrong, but I'd predict that horizontal row of gloss inputs is where a translator's eyes are going to spend the most time.

**Comment by arrocke** [comment]
Fixed the focus issue. I definitely agree that ideally, we would align the next link with the gloss input. The problem is that because the both the reference and target gloss fonts are configurable, the text baseline of the gloss input is variable. That's why I used a vertical line to separate them. At some point if we can know the text height of the different fonts we could do this, but for now, I was going to leave it like this.

**Comment by Pertempto** [comment]
That makes sense. If we aligned from the bottom, you'd only have to calculate based on the height of the gloss input, regardless of the reference height.

File: packages/web/src/features/translation/TranslationView.tsx [hunk 1]
```diff
@@ -331,6 +344,31 @@ export default function TranslationView() {
                   />
                 );
               })}
+              {canEdit && (
+                <li
+                  className={`
+                    mx-2 mb-4 flex items-center border-slate-300 h-28
```
Comment (Pertempto) [inline packages/web/src/features/translation/TranslationView.tsx]:
```suggestion
                    mx-2 mb-4 flex flex-col border-slate-300 h-28
```

File: packages/web/src/features/translation/TranslationView.tsx [hunk 2]
```diff
@@ -331,6 +344,31 @@ export default function TranslationView() {
...
+                    ${isHebrew ? 'border-r pr-4' : 'border-l pl-4'}
+                  `}
+                >
+                  <Button
+                    variant="tertiary"
+                    className="mb-2"
```
Comment (Pertempto) [inline packages/web/src/features/translation/TranslationView.tsx]:
```suggestion
                    className="mb-2 mt-auto"
```

**Review by Pertempto — APPROVED** [review]
The functionality works great. I added some suggestions that will align the next button to the bottom. It doesn't perfectly align with the gloss inputs, but I think it looks much more natural than aligning it vertically centered.

**Comment by arrocke** [comment]
I did some more digging and I think the line height issue was just in the select input in the language page, so I think for now we can just set a fixed top margin for alignment. We can come back to this later if we need to

**Review by Pertempto — APPROVED** [review]
That looks good! 

Just curious, what is the advantage of a bunch of top margin, as compared to just aligning to the bottom?

**Comment by arrocke** [comment]
You need spacing from the top or bottom either way. Aligning from the bottom requires an extra css property to set that alignment, and it also changes when the saving/approved indicator appears

**Pull request merged**